### DISCUSSION TITLE
Enter VR button not working

### DISCUSSION THREAD

**Discussion by dvdsmthdvd** [mozilla/hubs#4196] [Hubs Cloud Questions] [answered]
I am having trouble with a black screen when users click “enter VR” on Quest. This happens every time our users change avatar or name. It’s happened since the latest UI update, which installed for me last week. I see that there is a Github ticket, but don’t know how to fix this. Does anyone have suggestions for a solution? We have several research sessions planned this week and want to fix it as soon as possible.

https://github.com/mozilla/hubs/issues/4134

**Comment by netpro2k** [comment, marked as answer]
This should now be resolved by https://github.com/mozilla/hubs/pull/4206 - Note that you will need to manually pull this change into your hubs cloud branch if you want it immediately, otherwise it will be part of the next Hubs Cloud release.
